### ISSUE TITLE
Support changing uid/gid from the vinyl object to the file on disk

### ISSUE THREAD

**Issue by phated** [gulpjs/vinyl-fs#157]
We now allow `mtime`, `atime` and `mode` to be set on the actual file if they exist on the vinyl object.  We should also support `uid` and `gid` through use of `fs.fchown`.

Ref https://github.com/gulpjs/vinyl-fs/pull/151#issuecomment-183773006

**Comment by phated** [comment]
@erikkemperman @piranna would either of you have time to look into this?

**Comment by erikkemperman** [comment]
Probably not much, swamped at work.

For what it's worth, I expect this to be pretty easy to implement but annoying to test (cough-windows-cough).

**Comment by erikkemperman** [comment]
Sweet!

**Comment by pgnd** [comment]
@phated closed this in 9fd1689 on Jul 6, 2016

Looks like this was fixed.  However, I still see a chown fail

  https://github.com/sindresorhus/gulp-chown/issues/5#issuecomment-310393592

using gulp-chown.  That was originally pointed at vinyl-fs as the culprit, but I've installed vinyl-fs#master, and still see the problem.  Just checking in to see which thread to pull on -- gulp-chown, gulp-dest, or still vinyl-fs .., ?
